modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -46,6 +46,7 @@ def submit_form_2122 # rubocop:disable Metrics/MethodLength
             end
 
             power_of_attorney.save!
+            ClaimsApi::Logger.log('poa', poa_id: power_of_attorney.id, status: 'Created in Lighthouse')
```

Comment:
Your `case` for `poa` no longer works with a `status` attribute.
Instead, it's now a `detail` attribute.